build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -381,6 +389,25 @@ service ContentAddressableStorage {
     option (google.api.http) = { post: "/v2/{instance_name=**}/blobs:batchRead" body: "*" };
   }
 
+  // Create blobs by concatenating multiple smaller blobs.
+  //
+  // This method is only supported for digest function BLAKE3CONCAT. For
+  // other digest functions new blobs can only be created by uploading
+  // them in their entirety.
```

Comment:
Let me improve the phrasing here. The reason things are phrased the way they are, is because SplitBlobs() is documented as a call that *creates* blobs for you. And for the other hashing functions that can only be done by uploading.

But because this call is in practice used as a prerequisite for downloading objects, let me document it as such.